src/config/reader.rs

```diff
@@ -247,7 +246,7 @@ impl ConfigReader {
             }
```

Comment:
there is still `self.read_proto(import)` inside `while let`. How will that be handled for the reflection case?

Reply:
Not sure about that as of now. Maybe I’ll raise another pr if it can be divided into smaller part